.circleci/config.yml

```diff
@@ -0,0 +1,77 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2.1
+references:
+  workspace_root: &workspace_root
+    /tmp/workspace
+  attach_workspace: &attach_workspace
+    attach_workspace:
+      at: *workspace_root
+orbs:
+    aws-s3: circleci/aws-s3@1.0.0
+jobs:
+  canvasbuild:
+    docker:
+      - image: node:8.11.1
+    working_directory: /tmp/workspace
+    steps:
+      - *attach_workspace
+      - checkout
+      - run: rm -rf .git
+      - run: npm install -g uglifycss
+      - run: npm install -g uglify-js
+      - run: mkdir build
+      - run: cat js/*.js > build/theme.js
+      - run: uglifyjs build/theme.js > build/theme.min.js
+      - run: cat css/*.css > build/theme.css
+      - run: uglifycss build/theme.css > build/theme.min.css
+      - run: rm -rf node_modules
+      - persist_to_workspace:
+          root: *workspace_root
+          paths:
+            - ./
+  sendtoaws:
+    docker:
+      - image: circleci/python:2.7
+    working_directory: /tmp/workspace
+    steps:
+      - *attach_workspace
+      - run:
+          name: AWS Setup Environment Variables
+          command: |
+            echo 'export AWS_ACCESS_KEY_ID="$CANVAS_S3_AWS_KEY"' >> $BASH_ENV
+            echo 'export AWS_SECRET_ACCESS_KEY="$CANVAS_S3_AWS_SECRET"' >> $BASH_ENV
+            echo 'export AWS_REGION="us-east-1"' >> $BASH_ENV
+            source $BASH_ENV
+      - aws-s3/copy:
+          from: build/theme.min.css
+          to: 's3://hsph-canvas'
+      - aws-s3/copy:
+          from: build/theme.css
```

Comment:
Any reason why are moving the unpacked version?

Reply:
In case something goes wrong with the packing and we need them to see things. It cant hurt.